### ISSUE TITLE
Add e2e tests

### ISSUE THREAD

**Issue by sarah-cul** [cul-it/uls#1222]
Create e2e tests for the unit sites that are part of the the upstream. Create a separate test for each site using playwright. Playwright can be installed easily as an extension in VS Code. These tests can replace the cucumber tests that are part of the upstream currently. The goal is to have more thorough testing for all sites when the upstream is updated to make this updating process quicker and more stable.

**Comment by sarah-cul** [comment]
All unit sites now have playwright tests. They need to be run manually, but this works for now, saving time when deploying a new upstream to all the unit sites. Testing can be run on every environment (getupstream multidev branch, dev, test, and live).

https://github.com/cul-it/uls-playwright-e2e-tests

Incorporating these into the CI/CD to run automatically at various stages would be ideal. Need to do more research on a Pantheon feature called Quicksilver Platform Hooks "which allows you to run scripts after you perform actions such as code deployment".

**Comment by mhk33** [comment]
That's great! Would we be able to add playrwright tests to the main site?

**Comment by sarah-cul** [comment]
Good idea, we should discuss what tests make sense for the main site. I have the playwright extension installed in vscode where I can manually run them.

**Comment by mhk33** [comment]
Great, I'll make an issue! Would love to see how this works with vscode.
